### ISSUE TITLE
Fatal error on prepared statement patch

### ISSUE THREAD

**Issue by alessbelli** [plv8/plv8js#527]
Probably low priority, but just noticed that this minimal setup crashes the connection for any plv8 version I've tested it on.

```sql
DO $$
plv8.prepare('SELECT * FROM (VALUES (1)) AS foo').cursor().fetch.apply(null)
$$ language plv8;
```

error is
```
#
# Fatal error in v8::Object::GetInternalField()
# Internal field out of bounds
#

2023-04-07 18:37:09.277 UTC [1] LOG:  server process (PID 148) was terminated by signal 5: Trace/breakpoint trap
```

(this is very artificial as an example, but it came up trying to systematically patch the 3.0+ upgrade breaking change to BigInt - which otherwise are pretty hard to work with since they are non-json-serializable by default)

**Comment by JerrySievert** [comment]
thanks for the report!  crashes are not low priority, will take a look as soon as I get a chance.

**Comment by JerrySievert** [comment]
thank you very much for the fantastic reproduction case.  this has been fixed and v3.1.6 has been released addressing this issue.

I've also added `regressions.sql` to include this test and any future regression issues.
